packages/wormhole/src/consumer.ts

```diff
@@ -35,7 +35,10 @@ export const openWormhole = (
       detail: {
         consumer: this,
         fields: props,
-        updater: (prop, value) => { this[prop] = value; },
+        updater: (prop, value) => {
+          const target = (prop in el) ? el : this;
+          target[prop] = value;
+        },
```

Comment:
If the property exists in the element (if the component used `@Prop()`), set the value from the outside. Otherwise, set it internally (if the component used `@State()`)